lib/resources/r4/charge_item.yaml

```diff
@@ -129,8 +129,8 @@ fields:
       description: The national drug code for pharmacy charges. This binding is used in <code>national-drug-product</code> extensions.
       terminology:
         - display: National Drug Product
-          system: https://hl7.org/fhir/sid/ndc
-          info_link: https://hl7.org/fhir/sid/ndc
+          system: https://terminology.hl7.org/5.1.0/CodeSystem-v3-ndc.html
```

Comment:
it's ok to change the link, but the "system" needs to stay the same, as that is the official system of the NDC values